mindsdb/api/executor/datahub/datanodes/integration_datanode.py

```diff
@@ -138,22 +141,40 @@ def create_table(self, table_name: Identifier, result_set, is_replace=False, is_
         )
 
         try:
-            result = self.integration_handler.query(insert_ast)
+            result = self._query(insert_ast)
         except Exception as e:
             msg = f'[{self.ds_type}/{self.integration_name}]: {str(e)}'
             raise DBHandlerException(msg) from e
 
         if result.type == RESPONSE_TYPE.ERROR:
             raise Exception(result.error_message)
 
+    def _query(self, query):
+        time_before_query = time.perf_counter()
+        result = self.integration_handler.query(query)
```

Comment:
Maybe it would be useful to collect count of rows in result dataframe. If we want measure performance, we probably want to know how much data were processed during this period of time

Reply:
@tmichaeldb Maybe we can also measure the error rate?

Reply:
@tmichaeldb Maybe we an also measure the error rate?